tools/walletextension/rpcapi/blockchain_api.go

```diff
@@ -416,6 +388,48 @@ func extractCustomQueryAddress(params any) (*gethcommon.Address, error) {
 	return &address, nil
 }
 
+func extractSessionKeyAddress(params any) (*gethcommon.Address, error) {
```

Comment:
would be good to unite these 2 "extract" functions, as they're mostly similar. Or leave a todo